src/services/olHelpers.js

```diff
@@ -200,7 +200,7 @@ angular.module('openlayers-directive').factory('olHelpers', function($q, $log, $
                     $log.error('[AngularJS - Openlayers] - MapBox layer requires the map id and the access token');
                     return;
                 }
-                url = 'http://api.tiles.mapbox.com/v4/' + source.mapId + '/{z}/{x}/{y}.png?access_token=' +
+                url = 'https://api.mapbox.com/styles/v1/mapbox/' + source.mapId + '/tiles/256/{z}/{x}/{y}?access_token=' +
```

Comment:
line [226-228](https://github.com/jeffersonrpn/angular-openlayers-directive/blob/bece0b4c5f785f6adb160d25fe3f6a5434a3d7f5/src/services/olHelpers.js#L226) has the new Mapbox style. you could add the support for the tile Size if that's necessary. 
if you want to use the new MapBox: use `MapBoxStudio` as source type as stated in [220](https://github.com/jeffersonrpn/angular-openlayers-directive/blob/bece0b4c5f785f6adb160d25fe3f6a5434a3d7f5/src/services/olHelpers.js#L220)